error-constructors.js

```diff
@@ -0,0 +1,25 @@
+const list = [
+	// Native ES errors https://262.ecma-international.org/12.0/#sec-well-known-intrinsic-objects
+	EvalError,
+	RangeError,
+	ReferenceError,
+	SyntaxError,
+	TypeError,
+	URIError,
+
+	// Browser-specific errors
+	globalThis.DOMException,
```

Comment:
It exists in Node.js now too though.

Reply:
"Non-language errors"? The list before this one is part of ES itself, but `fetch` is just a Web API

Reply:
"Built-in errors"?